### ISSUE TITLE
Select element in filter component are not labelled

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#596]
The select element in the filter component:

![image](https://github.com/opencast/opencast-admin-interface/assets/2993/d1da9ff6-f7bf-4bd1-be9d-89b3c74442e5)

must have a label. This could be an implicit or explicit <label>, an aria-label or title attribute.

This has critical impact on blind/low vision people.
